workflow/workflow.py

```diff
@@ -1674,7 +1678,7 @@ def delete_paths(paths):
             return
 
         if isinstance(data, str):
-            data = bytearray(data)
+            data = bytearray(data, encoding="utf-8")
```

Comment:
This seems to be a Python3 thing

Reply:
Actually -> lets hold out a bit - I'm seeing about removing this code block all together... as it may not be required